processor/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
Does this need the copyright notice as well?

Reply:
added